examples/open-modal/src/app.js

```diff
@@ -32,8 +32,8 @@ import SavedMap from './components/saved-map';
 const MAPBOX_TOKEN = process.env.MapboxAccessToken; // eslint-disable-line
 
 class App extends Component {
-  componentWillReceiveProps(nextProps) {
-    if (!this.props.keplerGl.bar && nextProps.keplerGl.bar) {
+  componentDidUpdate(prevProps) {
+    if (!this.props.keplerGl.bar && prevProps.keplerGl.bar) {
```

Comment:
Looks like you broke the the logic here, since nextProps became prevProps it should be the other way around: `this.props.keplerGl.bar && !prevProps.keplerGl.bar`

Reply:
👍